Makefile

```diff
@@ -63,6 +63,9 @@ BUILD_DIR := ./build
 # Current version of the project.
 VERSION ?= $(shell git describe --tags --always --dirty)
 
+# Available cpus for compiling
```

Comment:
can we add the following issue link to provide necessary background?

https://github.com/caicloud/engineering/issues/8186

Reply:
https://github.com/caicloud/golang-template-project/pull/4 PTAL